assembl/static2/js/app/components/debate/thread/nuggets.jsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import range from 'lodash/range';
+
+import { getDomElementOffset, setDomElementOffset } from '../../../utils/globalFunctions';
+
+class Nuggets extends React.Component {
+  static get SPACER_SIZE() {
+    // Vertical space between overflowing extracts in px
+    return 100;
+  }
+
+  static completeLevelFromId(id) {
+    return id.split('-').slice(1).map((ident) => {
```

Comment:
Why do you use the DOM id?

Reply:
because I did not found a better way to find the preceding extract